aries-site/src/examples/components/button/ButtonBusyExample.js

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Button,
+  CheckBox,
+  Form,
+  FormField,
+  TextInput,
+  Page,
+  PageContent,
+} from 'grommet';
+import { ButtonGroup, ModalDialog } from 'aries-core';
+
+export const ButtonBusyExample = ({ containerRef }) => {
+  const [busy, setBusy] = useState(false);
+  const [success, setSuccess] = useState(false);
+  const [showLayer, setShowLayer] = useState(true);
+
+  const onClose = () => {
+    setShowLayer(false);
+    setSuccess(false);
+  };
+
+  return (
+    <Page fill>
+      <PageContent align="center" justify="center" fill>
+        <Button
+          alignSelf="center"
+          label="Reset example"
+          onClick={() => setShowLayer(true)}
+          secondary
+        />
+        {showLayer && (
+          <ModalDialog
+            title="Create cluster"
+            position="right"
+            full="vertical"
+            onClose={onClose}
+            target={containerRef?.current}
+          >
+            <Form
+              onValidate={() => {
+                // in production, replace timeout with
+                // conditionals based on API responses
+                setBusy(true);
+                setTimeout(() => {
+                  setBusy(false);
+                  setSuccess(true);
+                }, 2000);
+                setTimeout(() => {
+                  onClose();
+                }, 3000);
```

Comment:
If we add a check that success if false this prevents an issue where the button can be in the 'success' state and you can click on it again and both the success and busy state will show.
```suggestion
                if (success === false) {
                  setBusy(true);
                  setTimeout(() => {
                    setBusy(false);
                    setSuccess(true);
                  }, 2000);
                  setTimeout(() => {
                    onClose();
                  }, 3000);
                }
```